packages/kit/src/runtime/client/utils.js

```diff
@@ -98,24 +98,26 @@ export function find_anchor(element, base) {
 	/** @type {typeof valid_link_options['reload'][number] | null} */
 	let reload = null;
 
-	while (element !== document.documentElement) {
-		if (!a && element.nodeName.toUpperCase() === 'A') {
-			// SVG <a> elements have a lowercase name
-			a = /** @type {HTMLAnchorElement | SVGAElement} */ (element);
-		}
+	if (document.contains(element)) {
```

Comment:
style suggestion: do a `if (..) return` here instead, to safe some indentation (also I don't need to keep that if statement in my head as much)